Sources/Misc/DangerousSettings.swift

```diff
@@ -66,6 +66,12 @@ import Foundation
      */
     @objc public let customEntitlementComputation: Bool
 
+    /**
+     * Controls whether StoreKit 2 JWS tokens are sent to RevenueCat instead of StoreKit 1 receipts.
+     * Must be used in conjunction with the `usesStoreKit2IfAvailable configuration` option.
+     */
+    @objc public let usesStoreKit2JWS: Bool
+
```

Comment:
Eventually, we could merge this option with `usesStoreKit2IfAvailable` outright.